src/borg/archiver/rinfo_cmd.py

```diff
@@ -48,9 +46,27 @@ def do_rinfo(self, args, repository, manifest, cache):
                     location=repository._location.canonical_path(),
                     version=repository.version,
                     append_only=repository.append_only,
-                    **info,
+                    encryption=info["encryption"],
```

Comment:
guess `**info` would still work?

Reply:
It does, but that unpacks the entire dictionary and I think that's somewhat pointless if I only need one variable

Reply:
yeah, it is only one - until we pack more into it.